clients/src/main/java/org/apache/kafka/common/errors/IllegalClientTelemetryStateException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.errors;
+
+import org.apache.kafka.common.KafkaException;
+
+public class IllegalClientTelemetryStateException extends KafkaException {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
This seems unnecessary. Some KafkaExceptions seem to have it, while others do not. I can't see that having a serialVersionUID with a fixed value is helpful in this class.

Reply:
True not at all helpful, was following generally how other classes in same package are defined. Removed it.